frugalos_mds/src/node/node.rs

```diff
@@ -342,6 +342,10 @@ impl Node {
             Request::Stop => {
                 if self.phase == Phase::Running {
                     info!(self.logger, "Starts stopping the node");
+                    unsafe {
+                        self.rlog.io_mut().stop();
+                    }
+                    self.start_reelection();
```

Comment:
Stopping `rlog` before `start_election`  eliminates the possibility of electing a stopping node as a leader again.